src/pages/List.vue

```diff
@@ -6,12 +6,18 @@
 
 <script>
 import MonitorList from "../components/MonitorList.vue";
+import mobile from "../mixins/mobile";
 
 export default {
     components: {
         MonitorList,
     },
 };
+
+mobile.methods.onResize();
```

Comment:
Mixins are global for the entire app.
Interacting code should live there and not be run here as far as I understand vue.
https://vuejs.org/api/application.html#app-mixin